nntpit.c

```diff
@@ -525,6 +526,18 @@ void handle_list_cmd(client_t *cl, const char *param)
                               reddit_spool_lowwatermark(artmap));
         }
 
+        client_printf(cl, ".\r\n");
+        return;
+    } else if (strcasecmp(param, "OVERVIEW.FMT") == 0) {
+        client_printf(cl, "215 information follows\r\n");
+        // TODO: What are the field names usually used here?
+        client_printf(cl, "subject\r\n");
+        client_printf(cl, "author\r\n");
+        client_printf(cl, "date\r\n");
+        client_printf(cl, "message-id\r\n");
+        client_printf(cl, "references\r\n");
+        client_printf(cl, "byte-count\r\n");
+        client_printf(cl, "line-count\r\n");
```

Comment:
These should "bytes" and "lines", and then I guess it's safe to remove the TODO because these seem to be the correct field names!